DataTool/Models/ItemInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DataTool.JSON;
```

Comment:
I feel like `DataModels` would be a better name, people could think that Models is for 3d models e.g. skin models